### PR TITLE
kyverno: version stream

### PR DESCRIPTION
**Pull request by Dentrax**
VersionStream PR: https://github.com/chainguard-dev/package-version-metadata/pull/573

### CODE REVIEW

File: kyverno-1.12.yaml [hunk 1]
```diff
@@ -1,13 +1,15 @@
 package:
-  name: kyverno
+  name: kyverno-1.12
```
Comment (hectorj2f) [inline kyverno-1.12.yaml:2]: Could we check if we need to update any images using this package ?

Comment (hectorj2f) [reply kyverno-1.12.yaml:2]: Also we might want to copy the advisory of this new package rename too.

Comment (Dentrax) [reply kyverno-1.12.yaml:2]: > Could we check if we need to update any images using this package ?

I thought passing `provides:` would handle that:
```
provides:
       - kyverno=${{package.full-version}}
```

🤔

Comment (hectorj2f) [reply kyverno-1.12.yaml:2]: ops! yeah! never mind. But we need to copy the advisory.

Comment (Dentrax) [reply kyverno-1.12.yaml:2]: I may a bit confused here, should I `cp kyverno.advisories.yaml kyverno-1.12.advisories.yaml` (or `mv`?) and rename `package.name` to `kyverno-1.12`?

Comment (Dentrax) [reply kyverno-1.12.yaml:2]: https://github.com/wolfi-dev/advisories/pull/6585

**Review by imjasonh — DISMISSED** [review]

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**